pkg/config/config.go

```diff
@@ -49,6 +49,7 @@ const (
 	AnnotationMCOWithoutResourcesRequests = "mco-thanos-without-resources-requests"
 	AnnotationSkipCreation                = "skip-creation-if-exist"
 	AnnotationCertDuration                = "mco-cert-duration"
+	AnnotationProxyDisable                = "mco-proxy-disable"
```

Comment:
should we call it `mco-proxy-ignore` rather than disable? We are basically telling it to ignore global cluster level setting